ansible/roles/nginx/vars/couchdb2.yml

```diff
@@ -18,6 +18,8 @@ nginx_sites:
        balancer: "{{ deploy_env }}_couchdb_{{ couchdb2_proxy_port }}"
        proxy_redirect: ' off'
        client_max_body_size: ' 10m'
+       client_body_buffer_size: ' 512k'
```

Comment:
Needed for some puts/posts to couch

@millerdev I noticed some similar logs in the riak errors on nic. I assume the same is happening on prod/softlayer. Do you think it would be good to bump this up for riak as well? https://serverfault.com/questions/511789/nginx-client-request-body-is-buffered-to-a-temporary-file